package.json

```diff
@@ -7,12 +7,16 @@
   "scripts": {
     "lint": "eslint . --ext .js",
     "start": "web-dev-server --root-dir ./.vdiff --open ./.report/",
-    "test": "npm run lint && npm run test:server && npm run test:browser",
-    "test:browser": "web-test-runner --files \"./test/browser/**/*.test.js\" --node-resolve --playwright",
-    "test:server": "mocha ./test/server/**/*.test.js",
-    "test:vdiff": "web-test-runner --config ./test/browser/wtr-vdiff.config.js --group vdiff",
+    "test": "npm run lint && npm run test:bin && npm run test:server && npm run test:browser",
+    "test:bin": "mocha './test/bin/**/*.test.js'",
+    "test:browser": "web-test-runner --files './test/browser/**/*.test.js' --node-resolve --playwright",
+    "test:server": "mocha './test/server/**/*.test.js'",
+    "test:vdiff": "d2l-test --config ./test/browser/d2l-test.config.js --group vdiff",
```

Comment:
I still get a `d2l-test: command not found` when I run `npm run test:vdiff`. If I change this line to `npx` it works.

Reply:
Yeah it's been working for me because I have it linked which creates a global symlink, but we can add `npx` here. I kind of don't like it (maybe just aesthetically) but I didn't find any more official solution to this problem with a quick search so :shrug: